nixos/modules/services/web-apps/engelsystem.nix

```diff
@@ -99,7 +101,7 @@ in {
       '';
 
     services.phpfpm.pools.engelsystem = {
-      phpPackage = pkgs.php81;
+      phpPackage = phpExt;
```

Comment:
`pkgs.php` can be used directly all the extensions are already enabled by default (or if we want to keep it minimal, the `++ enabled` can be dropped and all the needed extensions should be added). Same for `pkgs/servers/web-apps/engelsystem/default.nix`.